keel-artifact/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/DebianArtifactSupplier.kt

```diff
@@ -62,32 +63,47 @@ class DebianArtifactSupplier(
 
   override fun getVersionDisplayName(artifact: PublishedArtifact): String {
     // TODO: Frigga and Rocket version parsing are not aligned. We should consolidate.
-    val appversion = AppVersion.parseName(artifact.version)
-    return if (appversion?.version != null) {
-      appversion.version
-    } else {
-      artifact.version.removePrefix("${artifact.name}-")
+    return try {
+      val appversion = AppVersion.parseName(artifact.version)
+      if (appversion?.version != null) {
+        return appversion.version
+      } else {
+        return artifact.version.removePrefix("${artifact.name}-")
+      }
+    } catch (ex: Exception) {
+      log.warn("trying to parse artifact ${artifact.name} with version ${artifact.version} but got an exception", ex)
+      Strings.EMPTY
```

Comment:
I think we should return `artifact.version` here, not whatever this value is, because it's for the display name

Reply:
done